das/dasrpc/dasRpcClient.go

```diff
@@ -51,9 +55,26 @@ func (clnt *DASRPCClient) Store(ctx context.Context, message []byte, timeout uin
 		Timeout:     response.Timeout,
 		SignersMask: response.SignersMask,
 		Sig:         sig,
+		KeysetHash:  keysetHash,
 	}, nil
 }
 
+func (clnt *DASRPCClient) KeysetFromHash(ctx context.Context, ksHash []byte) ([]byte, error) {
+	response, err := clnt.clnt.KeysetFromHash(ctx, &KeysetFromHashRequest{KsHash: ksHash})
+	if err != nil {
+		return nil, err
+	}
+	return response.Result, nil
```

Comment:
I think we'll want to retrieve this from on-chain, but if we do retrieve it over gRPC, we'll want to verify that the hash of these bytes matches the expected value.

Reply:
This is a general RPC client functionality, which is designed to return whatever the server returns.